docs/openapi.yaml

```diff
@@ -250,6 +255,18 @@ paths:
           schema:
             type: string
             example: 'owner'
+        - in: query
```

Comment:
nit: can use shared schema components

Reply:
was being lazy - added here! [8dcbd08](https://github.com/ar-io/arns-service/pull/76/commits/8dcbd0809fe585dc2fd8d04acbfbf768742175a7)

probably some patterns we can use in payment service (although we don't have duplicate params there for our endpoints)